### ISSUE TITLE
A11y_.NET Core_WinForms_PrintTesting_Print_ColorContrast: Color contrast ratio is less than 4.5:1 for the "Stores printing output to a file" text where the ratio is 3.278:1

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9248]
### .NET version

Microsoft.Net SDK6.0.311 (x64)

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.4.3;#DesktopApp;#Win11;#NonBenchmark;#ColorContrast

### Actual:
Color contrast ratio for "Stores printing output to a file" text 3.278:1 which is present below "Print to a file" where the ratio is less than 4.5:1.

### Expected:
Color contrast ratio for "Stores printing output to a file" text 3.278:1 which is present below "Print to a file" where the ratio should be greater than or equal to 4.5:1.

### User Impact:
Low vision users find it difficult to see the text if the color contrast ratio is less than 4.5:1.

### Attachment:
![Color Contrast](https://github.com/dotnet/winforms/assets/125462007/6f9611f9-ca1d-4e8e-9951-2146bbd579e8)

### Steps to reproduce

1. Install and Open "Winforms" App.
2. "Main Form" screen would appear.
3. TAB to "Printing_Controls:Testing the controls under Printing Tab" and press ENTER key.
4. TAB to "Print' button and press ENTER key.
5. Now "Printing from win32 application - Print" appears on the screen.
6. TAB to "Store Printing output to a file".
7.Open Accessibility Insights for Windows.
8.Now observe the issue that Color contrast ratio for "Stores printing output to a file" text 3.278:1 which is present below "Print to a file" where the ratio is less than 4.5:1.

**Comment by elachlan** [comment]
Duplicate of #9198?

**Comment by HindujaArugula** [comment]
Closing this bug as it is already logged. 
GitHubTags:#Closed;
